msteams-platform/graph-api/rsc/rsc-overview.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Overview resource-specific consent in Teams
+description: In this article, learn about resource-specific consent (RSC) permissions, types of RSC and supported RSC permissions.
+ms.localizationpriority: medium
+author: surbhigupta
+ms.author: surbhigupta
+ms.topic: conceptual
+---
+
+# Resource-specific consent for your Teams app
+
+Resource-specific consent (RSC) is an authorization framework built by Microsoft Teams and Microsoft identity platform that allows for granting scoped access to an app.
+
+Through RSC, an authorized user can give an app access to the data of a specific instance of a resource type. They don't need to give app access to every instance of the resource type in the entire tenant.
+
+For example, a person who owns both team A and team B can decide to give the data access for the Contoso app to only team A and not team B. The same concept of scoped data access applies to chats and meetings.
```

Comment:
For example, a person who owns both team A and team B can decide to give Contoso app access to the data of only team A and not team B.

Reply:
Updated as suggested.